src/commonTest/kotlin/com/github/michaelbull/result/BindingTest.kt

```diff
@@ -1,6 +1,6 @@
 package com.github.michaelbull.result
 
-import org.junit.Test
```

Comment:
Can we split this change up into a separate PR? This can be done in isolation to the general MPP migration and is a good change to have whether the mpp gets merged or not.

Reply:
sure thats a fair point!

Reply:
opened pr: https://github.com/michaelbull/kotlin-result/pull/19